src/non-standalone/Dialog/Types.ts

```diff
@@ -170,3 +171,66 @@ export interface IDialogConfigSign {
 	 */
 	setSignature?: (url: string) => void;
 }
+
+/**
+ * Object properties for image
+ */
+export interface ImageSelectorBoxProps {
+	/**
+	 * The base64 string of signature
+	 */
+	src: string;
+	/**
+	 * Boolean flag to set primary image
+	 */
+	setPrimary: boolean;
```

Comment:
this should be called `primary` not setPrimary, setPrimary sounds like a function